ansible/roles/debops.nginx/templates/etc/nginx/sites-available/php.conf.j2

```diff
@@ -11,7 +11,7 @@
 
 {% endblock %}
 {% block nginx_tpl_block_location_root %}
-                try_files $uri $uri/ $uri.php;
+                try_files {{ item.try_files|default("$uri $uri/ $uri.php") | join(' ') }};
```

Comment:
The idea is good, however I'm not sure if `join()` is needed here. Just expect a string, otherwise the generated line could have space between every letter since Python treats a string like a list of characters in this case.

Reply:
I followed the same pattern the current implementation used in `default.conf.j2` : https://github.com/debops/debops/blob/480482048a96e6758176ed38dd5fdbef318d659f/ansible/roles/debops.nginx/templates/etc/nginx/sites-available/default.conf.j2#L397

Changing this might lead in other inconsistencies, which can impact many other playbooks and users.

Reply:
Sounds good, thanks for clarification. :+1: